src/components/header/index.md

```diff
@@ -102,8 +102,6 @@ In November 2021, [the GOV.UK homepage introduced a menu bar](https://insidegovu
 <p>For example, a header link to “Get help with this service” must go to the same place as similar header links elsewhere in your service. This is to comply with WCAG 2.2 success criterion <a href="https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html">3.2.6 Consistent help</a>.</p>
 {% endcall %}
 
-Help links must always link to the same place. For example, a header link to “Get help with this service” must go to the same place as similar header links elsewhere in your service.
```

Comment:
As pointed out, these sentences were already incorporated into the paragraphs above before the WCAG 2.2 criteria link.